specs/commandline/sign.md

```diff
@@ -33,6 +33,7 @@ Flags:
   -h,  --help                       help for sign
        --id string                  key id (required if --plugin is set). This is mutually exclusive with the --key flag
   -k,  --key string                 signing key name, for a key previously added to notation's key list. This is mutually exclusive with the --id and --plugin flags
+       --oci-layout                 [Preview] whether the artifact is OCI image layout
```

Comment:
```suggestion
       --oci-layout                 [Preview] use OCI image layout for storage
```

Reply:
Suggestion: Sign the artifact stored in the OCI image layout